docs/concepts/intro.rst

```diff
@@ -0,0 +1,41 @@
+.. meta::
+  :description: rocPRIM documentation and API reference library
+  :keywords: rocPRIM, ROCm, API, documentation
+
+.. _rocprim-intro:
+
+********************************************************************
+ Introduction to rocPRIM
+********************************************************************
+
+
+Operations and Sequences
+========================
+
+A rocPRIM operation is a computation over a sequence of objects returning one value like ``reduce``, returning another sequence like ``sort``, or multiple sequences like ``partition``. The elements of the sequence could be of any type or class, although template specialization allows rocPRIM to optimize the computations over the usual numerical datatypes. Operations handle sequences by expecting ``iterators`` as input and mutable ones as output.
+
+A high level view of the available operations can be found on :ref:`ops-summary`. rocPRIM includes a variety of generic operations that are frequently very useful.
+
+.. note::
+  Refer to :ref:`data-type-support` for information on supported datatypes.
+
+Scope
+======
+
+An important property of a rocPRIM operation is its scope defining at which level of the computing model the processing will take place. That means which parts of the GPU will cooperate to compute the result.
```

Comment:
Just an idea - splitting the first sentence in two and adding a connecting phase before the last sentence might help make this easier to read. For example, 
"An important property of a rocPRIM operation is its scope. An operation's scope determines the level of the computing model that will be used to perform the processing. In other words, scope determines which parts of the GPU will cooperate to compute the result."

Reply:
Fixed as suggested.